### PR TITLE
WebUI: 2328

### LINKED ISSUE #2328: WebUI: Fade should not depend on current VoiceGroup
Mir ist aufgefallen, dass es bisher auch eine Wirkung vom Part-Selektor auf den Fade-Range- bzw. Split-Point-Editor gibt. Das bitte auch rausnehmen, falls Du es nicht schon gemacht hast. Reinnehmen würde ich hingegen, dass bei Bedienung eines der Wertefelder neben den Editoren die jeweilige Farbe nach vorne kommt.

### PR DESCRIPTION
**Pull request by schlenkibus**
made fade editor react to last changed parameters: color gets put in front. Removed Selected Anfasser drawing (outline). closes #2328

### CODE REVIEW

File: projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java [hunk 1]
```diff
@@ -38,6 +37,31 @@ public KeyBed(Control parent) {
             invalidate(INVALIDATION_FLAG_UI_CHANGED);
             return true;
         });
+
```
Comment (hhoegelo) [inline projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java:40]: Consider enhancing and then using the EditBufferPresenter.

Comment (schlenkibus) [reply projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java:40]: Do you mean move the contents of the new FadeEditorPresenter into the existing EditBufferPresenter?

Comment (hhoegelo) [reply projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java:40]: No, just meant to not use EditBufferModel in UI code.

File: projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java [hunk 2]
```diff
@@ -393,4 +396,21 @@ public Control mouseDrag(Position oldPoint, Position newPoint, boolean fine) {
         }
         return super.mouseDrag(oldPoint, newPoint, fine);
     }
-}
+
+    @Override
+    public Control mouseDown(Position p) {
+        mouseIsDown = true;
```
Comment (hhoegelo) [inline projects/epc/web-ui/src/main/java/com/nonlinearlabs/client/world/overlay/belt/fadeeditor/KeyBed.java:402]: Don't we need invalidations whenever the mouseIsDown state changes?

**Review by hhoegelo — APPROVED** [review]

**Pull request merged**